### PR TITLE
Add get_keras_model to TensorflowModel

### PR DESCRIPTION
**Pull request by mathilde-leval**
Adds the possibility to build a Keras model from the underlying saved model in the TensorflowModel class.

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 @nox.session(python=["3.8", "3.9", "3.10", "3.11"])
 def test(session):
     # Install deps and the package itself.
-    session.install("-r", "requirements-dev.txt")
+    session.install("-r", "requirements-optional.txt")
```
[the remark this answers is not in the document]
Comment (mathilde-leval) [reply noxfile.py]: Looks like the CI is not happy

**Comment by tbascoul** [comment]
Nice solution 👍

**Review by tbascoul — APPROVED** [review]

**Review by tbascoul — APPROVED** [review]

**Review by antoinejeannot — APPROVED** [review]

**Comment by antoinejeannot** [comment]
Hi @mathilde-leval 👋 
I've finally had some time to work on your PR. 
Could you please take a look, and if the changes suit you, I'll let you squash and merge 👐

**Comment by mathilde-leval** [comment]
Hi @antoinejeannot thanks for your time :) 
Oh I see, nice solution indeed. LGTM!

**Pull request merged**